decisions/flow_versions.md

```diff
@@ -0,0 +1,11 @@
+# Flow versions
+
+## Decision
+The version of `flow-bin` that we use in Backpack React Native should align to the same version that React Native is using.
```

Comment:
```suggestion
The version of `flow-bin` that we use in Backpack React Native should align to the [same version that React Native is using](https://github.com/facebook/react-native/blob/master/.flowconfig).
```

Adding a link just to make it easier for future backpackers to find it

Reply:
Like this!